functors_task_1/student/include/impl.h

```diff
@@ -16,16 +46,18 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(BeerBrand beer)
```

Comment:
please mark function "in" params const by default, because their modification if considered as a bad practice

Reply:
I don't understand. What do you mean exactly?

Reply:
Please change param to const BeerBrand - use const by default